src/index/Vocabulary.h

```diff
@@ -306,3 +306,18 @@ class Vocabulary {
 
 using RdfsVocabulary = Vocabulary<CompressedString, TripleComponentComparator>;
 using TextVocabulary = Vocabulary<std::string, SimpleStringComparator>;
+using TextVocabulary = Vocabulary<std::string, SimpleStringComparator>;
+
```

Comment:
Add an empty comment line ` // ______________________________________________________________` before that definition to make it visible.